HW_5/main.py

```diff
@@ -0,0 +1,49 @@
+from pydantic import BaseModel
+from fastapi import FastAPI
+from typing import Dict, List
+from collections import defaultdict
+
+
+app = FastAPI()
+
+
+class Task(BaseModel):
+    id: int
+    title: str
+    description: str
+    status: bool = False
+
+
+tasks_db: Dict[int, Task] = defaultdict()
+
+
+@app.get("/tasks")
+async def read_tasks(skip: int = 0, limit: int = 100):
+    return list(tasks_db.values())[skip: skip + limit]
+
+
+@app.get("/tasks/{task_id}", response_model=Task)
+async def read_task(task_id: int):
+    return tasks_db[task_id]
+
+
+@app.post("/tasks", response_model=Task)
+async def create_task(task: Task):
+    task_id = len(tasks_db) + 1
+    db_task = Task(id=task_id, **task.dict())
+    tasks_db[task_id] = db_task
+    return db_task
```

Comment:
Вы разве не тестировали ваши конечные точки через `Swagger`? Просто здесь явная ошибка, а именно в 33 строке. Вы сначала добавляете именной аргумент `id`, а потом распаковывайте словарь с `task`, где уже есть этот аргумент. В результате чего возникает ошибка: `TypeError: sample.Task() got multiple values for keyword argument 'id'`. Нужно сделать так
```suggestion
    task_id = len(tasks_db) + 1
    task.id = task_id
    tasks_db[task_id] = task
    return task
```